clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +273,53 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStoreStatistics(ServerStoreImpl store, String storeName) throws InvalidStoreException {
+    STAT_STORE_METHOD_REFERENCES.entrySet().stream().forEach((entry)->
+      registerStatistic(store, storeName, entry.getKey(), STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
```

Comment:
I am actually passing that Callable to the registerStatistic(...) method which then uses it to call:
StatisticsManager.createPassThroughStatistic(context, observerName, tags, properties, callable);
So I am pretty sure I need it, unless there is some magic I am not understanding with streams.